Sources/Fullscreen/TransactionView/TransactionStepView/TransactionStepView+Style.swift

```diff
@@ -38,9 +38,9 @@ public extension TransactionStepView {
         var titleTextColor: UIColor {
             switch self {
             case .notStarted:
-                return .stone
+                return .textSecondary
             default:
```

Comment:
Shouldn't this one be exhaustive as well?

Reply:
I see the ones for `bodyTextColor` and `detailTextColor` list all cases

Reply:
The title stays the same regardless of state (atleast for now) 😄

Reply:
The only special case is if it's `notStarted`